libnetwork/internal/util/parse.go

```diff
@@ -34,3 +34,16 @@ func ParseVlan(vlan string) (int, error) {
 	}
 	return v, nil
 }
+
+// ParseNoAutoGateway parses the no_auto_gateway option
+func ParseNoAutoGateway(opt string) (bool, error) {
+	switch opt {
```

Comment:
you should use strconv.ParseBool() instead.

Reply:
Done